remote-config/USERGUIDE.md

```diff
@@ -37,10 +43,12 @@ The Remote Config SDK is configured via manifest meta-data, the configurable val
 | RAS Application ID           | String  | `com.rakuten.tech.mobile.ras.AppId`                  | ❌         | 🚫        |
 | RAS Project Subscription Key | String  | `com.rakuten.tech.mobile.ras.ProjectSubscriptionKey` | ❌         | 🚫        |
 | Config Application           | boolean | `com.rakuten.tech.mobile.remoteconfig.ApplyDirectly` | ✅         | `false`   |
+| Polling Delay (in secs)      | integer | `com.rakuten.tech.mobile.remoteconfig.PollingDelay`  | ✅         | `3600`    |
 
 Notes:
 * We don't currently host a public API, so you will need to provide your own Base URL for API requests.
 * If Config Application is set to `true`, config are applied directly when fetched. Otherwise, fetched config are applied on next app launch from terminated state.
+* By default, config values are fetched from the API at launch time and then periodically fetched again every 60 minutes. Set the Polling delay to the desired interval of fetching config values.
```

Comment:
This should also mention that the minimum polling delay is 60 seconds.

Reply:
added note